README.md

```diff
@@ -3,3 +3,44 @@
 <img src="https://media.giphy.com/media/143vPc6b08locw/giphy.gif" width="300">
 
 _Ship-it_ is Wattpad's tool for continuously deploying code.
+
+## Local Development
+
+This project uses skaffold for local development and testing. To get started,
+you'll need to install these tools locally:
+
+* helm: https://github.com/helm/helm
+* minikube: https://github.com/kubernetes/minikube
+* skaffold: https://github.com/GoogleContainerTools/skaffold
+
+1. Create a new minikube cluster
+
+```bash
+$ minikube create
+```
+
+2. Update kubectl's cluster context
+
+```bash
+$ minikube update-content
+$ kubectl config current-context
+minikube
+```
+
+3. Install tiller in minikube
+
+```bash
+$ helm init
+```
+
+4. Run skaffold in development mode
```

Comment:
Before running `skaffold dev, the custom resource will need to be installed on the local cluster by running `kubectl apply -f deploy/v1alpha1_helmrelease_crd.yaml`.

Reply:
We'll fix this by including the CRD in the `ship-it` chart